pushy/src/main/java/com/relayrides/pushy/apns/ApnsClientHandler.java

```diff
@@ -361,7 +361,7 @@ public void operationComplete(final ChannelPromise future) throws Exception {
     @Override
     public void userEventTriggered(final ChannelHandlerContext context, final Object event) throws Exception {
         if (event instanceof IdleStateEvent) {
-            assert PING_TIMEOUT < ApnsClient.PING_IDLE_TIME_MILLIS;
+            assert PING_TIMEOUT < this.apnsClient.idlePingIntervalMillis;
```

Comment:
@jchambers What should be the default value of PING_TIMEOUT? Currently, this assert will always pass since its comparing seconds < milliSeconds